### PR TITLE
[3.1.0] Fix and sync config catalog inconsistencies

### PR DESCRIPTION
**Pull request by RakhithaRR**
## Purpose
This PR fixes the inconsistencies in the config catalog related files for the master branch.

- Added missing configs
- Synced files in en/docs/reference and en/tools
- Added sample .toml files for all configs
- Updated the script template files

Related doc issue: https://github.com/wso2/docs-apim/issues/2383

### CODE REVIEW

**Review by Mariangela — APPROVED** [review]

File: en/docs/reference/config-catalog.md [hunk 1]
```diff
@@ -725,193 +700,176 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
...
                                             <span class="param-default-value">Default: <code></code></span>
                                         </div>
-
+                                        
                                     </div>
                                     <div class="param-description">
-                                        <p>You can define the custom claim retiriver implmentation by adding the
-                                            following
-                                            apim.jwt.enable_user_claims=true.
+                                        <p>You can define the custom claim retiriver implmentation by adding the following
```
Comment (hisanhunais) [inline en/docs/reference/config-catalog.md:833]: Shall we fix these?
"retriever"
"implementation"

File: en/tools/config-catalog-generator/data/apim.analytics.url_group.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#LOADBALANCER CONFIGS
+[[apim.analytics.url_group]]
+analytics_url =["tcp://analytics1:7611","tcp://analytics2:7611"]
```
Comment (hisanhunais) [inline en/tools/config-catalog-generator/data/apim.analytics.url_group.toml:3]: Shall we add a space after the "=" sign in all of the configs in this file?

File: en/docs/reference/config-catalog.md (hunk 1 above)
Comment (RakhithaRR) [reply en/docs/reference/config-catalog.md:833]: Fixed with https://github.com/wso2/docs-apim/pull/3004

File: en/tools/config-catalog-generator/data/apim.analytics.url_group.toml (hunk 2 above)
Comment (RakhithaRR) [reply en/tools/config-catalog-generator/data/apim.analytics.url_group.toml:3]: Fixed with https://github.com/wso2/docs-apim/pull/3004

**Pull request merged**